npm/ng-packs/packages/theme-shared/src/lib/directives/ngx-datatable-list.directive.ts

```diff
@@ -53,14 +53,6 @@ export class NgxDatatableListDirective implements OnChanges, OnDestroy, OnInit {
     };
   }
 
-  private subscribeToPage() {
-    const sub = this.table.page.subscribe(({ offset }) => {
-      this.list.page = offset;
-      this.table.offset = offset;
-    });
-    this.subscription.add(sub);
-  }
-
```

Comment:
This change causing broken pagination now. Whether the `_filter` is empty or not, pagination does not work on the `ngx-datatable` component.

## Reproduce
Just try to paginate any `ngx-datatable` that works with `[list]`, and nothing happens.

@Sinan997 @oykuermann

Reply:
We have moved pagination functionality to ngx-datatable's `input, output`.
https://github.com/abpframework/abp/pull/19203/files#diff-0c5913f6f68aa1645a0b5aa522f62c0b7e5f48657e8398c2f625926b138f847dR6-R7

So if you are using ngx-datatable directly. You should add the code below to your ngx-datatable element

```ts
[offset]="list?.page"
(page)="list.page = $event.offset"
```

Reply:
I think a patch version (8.0.x) should not contain a breaking change. I'm sure you can find a way that supports both.